packages/lit-dev-content/site/docs/tools/publishing.md

```diff
@@ -106,16 +106,26 @@ You can run Babel via a bundler plugin such as [@rollup/plugin-babel](https://ww
 
 The following are other good practices to follow when publishing reusable Web Components.
 
-### Do not import polyfills into modules
+### Don't import polyfills into modules
 
 Polyfills are an application concern, so the application should depend directly
 on them, not individual packages. The exact polyfills needed often depends on
 the browsers the application needs to support, and that choice is best left to
-the application developer using your component.
+the application developer using your component. Your component's documentation **should** clearly identify any APIs it uses that may require polyfills. 
 
 Packages may need to depend on polyfills for tests and demos, so if
 they're needed, they should only go in `devDependencies`.
 
+### Don't bundle, minify, or optimize modules
+
+Bundling and other optimizations are application concerns. Bundling can also introduce multiple versions of Lit (and other packages) into your application since npm can't deduplicate the packages. This causes bloat and may cause bugs.
+
+Optimizing modules before publication may also prevent application-level optimizations.
+
+Bundling and other optimizations can be valuable when serving a module from a CDN. Because of the potential issues with bundling, this should usually be used for prototyping, not for production. 
```

Comment:
There's something about these two sentences that almost read as contradictory. Bundling is valuable. Bundling has issues. Feels like we want some more nuance.

```suggestion
Bundling and other optimizations can be valuable when serving a module from a CDN, but since users may need to use multiple packages that depend on Lit, serving from a CDN can result in users loading more code than necessary. For these reasons we recommend performance-sensitive applications always build from npm where packages can be deduplicated, rather than loading bundled packages off of a CDN.
```